test/e2e/common/configmap_volume.go

```diff
@@ -90,21 +89,20 @@ var _ = ginkgo.Describe("[sig-storage] ConfigMap", func() {
 	*/
 	framework.ConformanceIt("should be consumable from pods in volume with mappings and Item mode set [LinuxOnly] [NodeConformance]", func() {
 		mode := int32(0400)
-		doConfigMapE2EWithMappings(f, 0, 0, &mode)
+		doConfigMapE2EWithMappings(f, false, 0, &mode)
 	})
 
 	/*
 		Release : v1.9
 		Testname: ConfigMap Volume, with mapping, non-root user
 		Description: Create a ConfigMap, create a Pod that mounts a volume and populates the volume with data stored in the ConfigMap. Files are mapped to a path in the volume. Pod is run as a non-root user with uid=1000. The ConfigMap that is created MUST be accessible to read from the newly created Pod using the volume mount. The file on the volume MUST have file mode set to default value of 0x644.
-		This test is marked LinuxOnly since Windows does not support running as UID / GID.
 	*/
-	framework.ConformanceIt("should be consumable from pods in volume with mappings as non-root [LinuxOnly] [NodeConformance]", func() {
-		doConfigMapE2EWithMappings(f, 1000, 0, nil)
+	framework.ConformanceIt("should be consumable from pods in volume with mappings as non-root [NodeConformance]", func() {
```

Comment:
Isn't "RunAsUserName" still an alpha feature and is gated? With the `LinuxOnly` tag removed, we'd be running these tests against windows nodes and they'd fail if the feature is gated (i.e., the default).

Reply:
True, which is why I was saying that we should add ``--enable-admission-plugins=WindowsRunAsUserName`` on kube-apiserver controllers in our runs.

Note that this change will only affect the master branch (and probably the release-1.16 branch too).

Reply:
I don't think it's a good idea to force all the test jobs to run with an alpha feature enabled for a conformance test. I'd prefer keeping the `[LinuxOnly]` tag until the feature has been promoted to beta.

In the meantime, we can copy the test(s) and add `[Feature:Windows]` if we want the coverage for the alpha feature.

Reply:
Ok, I'll leave only the basic RunAsUserName tests for now, and I'll resubmit this part when the features goes to Beta.